.claude/metadata-updater.sh

```
@@ -142,9 +142,8 @@ update_metadata_key() {
   # Create temp file
   local temp_file="${metadata_file}.tmp"
 
-  # Escape special sed characters in value (&, \, and | since | is our delimiter)
-  local escaped_value
-  escaped_value=$(printf '%s' "$value" | sed 's/[&|\\]/\\&/g')
+  # Escape special sed characters in value (&, \, and | — all special with | delimiter)
+  local escaped_value=$(echo "$value" | sed 's/[&|\\]/\\&/g')
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)